lib/cmd/license-file-handler.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+
+class LicenseFileHandler
+{
+  static getFile(filename)
+  {
+    return new Promise((resolve,reject) => {
+      if (fs.existsSync(filename))
+        return resolve(fs.readFileSync(filename, 'utf8'));
+      else
+        return reject(
+          {
+            'errorCode': 'LICENSE_CONFIG_NOT_FOUND',
+            'errorMessage': 'Could not find any license config file.'
+          });
+    });
+  }
+
+  static parseFile(arg)
+  {
+    return new Promise((resolve,reject) => {
+      let object = null;
+      
+      try {
+        object = JSON.parse(arg);
+      } catch (e) {
+        return reject(
+          {
+            'errorCode': 'LICENSE_CONFIG_CORRUPTED',
+            'errorMessage': 'License config file is not a valid JSON.'
+          });
+      }
+
+      if (object.header !== null && object.header !== undefined &&
```

Comment:
Also, the best approach would be a clear error message. As:
```
isValidJsonObject(object){
  let missingFields = [];
  if(object.file === null || object.file === undefined){
    missingFields.push('file');
  }
  //...(for each field)
  if(missingFields.length > 0){
    promise.reject( {
             'errorCode': 'LICENSE_CONFIG_INVALID',
             'errorMessage': `Missing mandatory fields in License config: ${missingFields.split()}`,
             'errorFields': missingFields
} .`
    });
  }
}
```

If it is ok for you, you also can change validation from if("object.file === null || object.file === undefined") to if(object.file). It will validate for empty, null, undefined, false and 0.